app/bundles/CoreBundle/Factory/MauticFactory.php

```diff
@@ -329,7 +329,7 @@ public function getInstalledThemes($specificFeature = 'all', $extended = false)
     }
 
     /**
-     * Returns MailHelper wrapper for Swift_Message via $helper->message.
+     * Returns MailHelper wrapper for Email via $helper->message.
```

Comment:
@escopecz I read in the code that `MauticFactory` should be removed, I need some support in removing from this bundle.

Reply:
Let's do that once this PR is merged. Thanks for thinking about this.